sdk/trace/span.go

```diff
@@ -614,10 +619,10 @@ func startSpanInternal(ctx context.Context, tr *tracer, name string, o *trace.Sp
 
 	startTime := o.Timestamp()
 	if startTime.IsZero() {
-		startTime = time.Now()
+		span.stopwatch = tr.provider.clock.Stopwatch()
+	} else {
+		span.stopwatch = standardStopwatch(startTime)
```

Comment:
If the user supplies their own start time it shouldn't mean the user supplied clock is not used. This seems like a design flaw.

Should we updated the `Clock` interface's `Stopwatch` method to accept a start time?